EIPS/eip-7044.md

```diff
@@ -0,0 +1,57 @@
+---
+eip: 7044
+title: Perpetually Valid Signed Voluntary Exits
+description: Lock voluntary exit signature domain on capella for perpetual validity
+author: Lion (@dapplion)
+discussions-to: https://github.com/ethereum/consensus-specs/pull/3288
+status: Draft
+type: Standards Track
+category: Core
+created: 2023-05-18
+---
+
+## Abstract
+
+Lock validator voluntary exit signature domain on Capella for perpetual validity. Currently, signed voluntary exits are only valid for two forks.
+
+## Motivation
+
+Currently, signed voluntary exits are valid for only two forks, due to the Beacon Chain state holding only the current and previous fork. This limitation affects increases the complexity of some staking operations. Specifically those that require the exchange of signed exits ahead of time valid for an unbounded number of forks.
+
+The limited validity of volutary exits was originally motivated to isolate them in the event of a perpetual fork. If fork A and B exist and a validator operators on both, if they send an exit, it will be replayable on both. However, this positive is not sufficient to justify the UX degradation exposed above.
```

Comment:
```suggestion
The limited validity of voluntary exits was originally motivated to isolate them in the event of a perpetual fork. If fork A and B exist and a validator operates on both, if they send an exit, it will be replayable on both. However, this possibility is not sufficient to justify the UX degradation exposed above.
```